### PR TITLE
Mark RotateKubeletClientCertificate GA

### PR DESCRIPTION
**Pull request by tengqm**
closes: #21696

### CODE REVIEW

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 1]
```diff
@@ -122,7 +122,6 @@ different Kubernetes components.
 | `QOSReserved` | `false` | Alpha | 1.11 | |
 | `RemainingItemCount` | `false` | Alpha | 1.15 | |
 | `ResourceLimitsPriorityFunction` | `false` | Alpha | 1.9 | |
-| `RotateKubeletClientCertificate` | `true` | Beta | 1.8 | |
 | `RotateKubeletServerCertificate` | `false` | Alpha | 1.7 | 1.11 |
```
Comment (sftim) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:125]: You also need to move this line, right?

Comment (sftim) [reply content/en/docs/reference/command-line-tools-reference/feature-gates.md:125]: Ah no. But we _should_ have a line for when *RotateKubeletClientCertificate* was alpha. Unless it went insta-beta.

Comment (tengqm) [reply content/en/docs/reference/command-line-tools-reference/feature-gates.md:125]: 1.8 was a long history to dig back. We have no idea whether there was an Alpha at all. Even if there WAS one, what do we gain by adding a line for pre-1.8 release? We are not longer supporting it, right?

Comment (sftim) [reply content/en/docs/reference/command-line-tools-reference/feature-gates.md:125]: It's a minor thing for sure.

**Comment by sftim** [comment]
This change on its own isn't wrong; however, to do all the work we also need to revise `content/en/docs/tasks/tls/certificate-rotation.md` to drop mention of the RotateKubeletClientCertificate feature gate.
@tengqm how do you want to move forward on that?

**Comment by tengqm** [comment]
@sftim The mentioned page has been taken care of by liggit (#21108)

**Pull request merged**